src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -293,6 +289,85 @@ export class StudyViewPageStore {
 
     @observable private chartsType = observable.map<ChartType>();
 
+    @action
+    updateStoreFromURL(query: any) {
+        let newStudyIdsString;
+        if ('studyId' in query) {
+            newStudyIdsString = (query.studyId as string);
+        }
+        if ('id' in query) {
+            newStudyIdsString = (query.id as string);
+        }
+        if (newStudyIdsString) {
+            const newStudyIds = newStudyIdsString.trim().split(",");
+            if (!_.isEqual(newStudyIds, this.studyIds)) {
+                // update if different
+                this.studyIds = newStudyIds;
+            }
+        }
+        let filters: Partial<StudyViewFilter> = {};
+        if ('filters' in query) {
+            filters = JSON.parse(decodeURIComponent(query.filters)) as Partial<StudyViewFilter>;
+        }
+
+        if (filters.hasOwnProperty('clinicalDataEqualityFilters') && filters.clinicalDataEqualityFilters!.length > 0) {
```

Comment:
i've realized hasOwnProperty is not a good idea because typescript.  might as well just test property directly (filters.clinicalDataEqualityFilters)